packages/mask/src/plugins/Profile/constants.ts

```diff
@@ -3,3 +3,7 @@ export const PLUGIN_NAME = 'Profile'
 export const PLUGIN_DESCRIPTION = 'Profile'
 
 export const RSS3_DEFAULT_IMAGE = 'https://rss3.mypinata.cloud/ipfs/QmVFq9qimnudPcs6QkQv8ZVEsvwD3aqETHWtS5yXgdbYY5'
+
+export const RSS3_PROFILE_URL_RE = /^https?:\/\/(\w+)\.rss3\.bio/
```

Comment:
suggest, use named groups?

```suggestion
export const RSS3_PROFILE_URL_RE = /^https?:\/\/(?<name>\w+)\.rss3\.bio/
```

```
> pattern = /^https?:\/\/(?<name>\w+)\.rss3\.bio/
/^https?:\/\/(?<name>\w+)\.rss3\.bio/
> matched = pattern.exec('https://cz_binance.rss3.bio/')
[
  'https://cz_binance.rss3.bio',
  'cz_binance',
  index: 0,
  input: 'https://cz_binance.rss3.bio/',
  groups: [Object: null prototype] { name: 'cz_binance' }
]
> matched.groups.name
'cz_binance'
```

Reply:
iOS doesn't support regexp group yet.

Reply:
> iOS doesn't support regexp group yet.

Safari 15.2 supported

Reply:
![image](https://user-images.githubusercontent.com/5390719/146718892-7b41332f-c937-4f9f-9ec8-e3e735e16ff0.png)

All green according to https://kangax.github.io/compat-table/es2016plus/